Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -193,57 +208,47 @@ public void HandleKeyPress(KeyPress keyPress)
-            BaseShape shape2 = shapeFactory.GetShape(Shape.DamageCrystal, 5, 15);
-            BaseShape shape3 = shapeFactory.GetShape(Shape.Saw, 8, 15);
-            GamePlatforms.AddShape(shape1);
-            GamePlatforms.AddShape(shape2);
-            GamePlatforms.AddShape(shape3);*/
-
-            /* ABSTRACT SHAPE FACTORY DESIGN PATTERN */
-            /*AbstractFactory abstractShapeFactory = FactoryProducer.GetFactory(passable: true);
-            BaseShape aShape1 = abstractShapeFactory.CreateStaticShape(Shape.Platform, 0, 0, 0, 10);
-            BaseShape aShape2 = abstractShapeFactory.CreateEntityShape(Shape.HealthCrystal, 10, 15);
-            BaseShape aShape3 = abstractShapeFactory.CreateEntityShape(Shape.DamageCrystal, 15, 15);
-            GamePlatforms.AddShape(aShape1);
-            GamePlatforms.AddShape(aShape2);
-            GamePlatforms.AddShape(aShape3);*/
-
-            AbstractFactory impassableFactory = FactoryProducer.GetFactory(passable: false);
-            BaseShape firstPlatform = impassableFactory.CreateStaticShape(Shape.Platform,
-                new CombinedGenerationStrategy(), 0, 0, GameSettings.HorizontalCellCount,
-                GameSettings.VerticalCellCount / 3);
-            PlatformsContainer.AddShape(firstPlatform);
-
-            AbstractFactory passableFactory = FactoryProducer.GetFactory(passable: true);
-            BaseShape secondPlatform = passableFactory.CreateStaticShape(Shape.Platform,
-                new PlatformerGenerationStrategy(), 0, GameSettings.VerticalCellCount / 3 + 1,
-                GameSettings.HorizontalCellCount, GameSettings.VerticalCellCount * 2 / 3);
-            PlatformsContainer.AddShape(secondPlatform);
-
-            // Generate the player above the first platform
-            Player = new Player(secondPlatform.CenterPosX, secondPlatform.CenterPosY + 1);
-
-            BaseShape thirdPlatform = passableFactory.CreateStaticShape(Shape.Platform,
-                new RandomlySegmentedGenerationStrategy(), 0, GameSettings.VerticalCellCount * 2 / 3 + 1,
-                GameSettings.HorizontalCellCount, GameSettings.VerticalCellCount - 3);
-            PlatformsContainer.AddShape(thirdPlatform);
-
-            var coordinates = firstPlatform.GetCoords();
-            int[] xCoords = coordinates.Item1;
-            int[] yCoords = coordinates.Item2;
-            int randomLocation = RandLocation(xCoords, yCoords);
-            PowerUp testPowerUp =
-                new PowerUp(xCoords[randomLocation], yCoords[randomLocation] + 2, PowerUps.Double_Jump);
-            PowerUpsContainer.AddShape(testPowerUp);
+            var gameeStateBuilder = new GameStateBuilder();
+            var initialGameState = gameeStateBuilder.Configure()
+                .InitializeMap(GameSettings.HorizontalCellCount, GameSettings.VerticalCellCount)
+                .InitializePowerUps(GameSettings.HorizontalCellCount, GameSettings.VerticalCellCount)
```

Comment:
We need to somehow show that the pattern impls work. There isn't really a good way to show this visually, so maybe add logging to the builder steps?